src/app/modules/group/components/group-log-view/group-log-view.component.ts

```diff
@@ -19,22 +31,35 @@ interface Data {
   templateUrl: './group-log-view.component.html',
   styleUrls: [ './group-log-view.component.scss' ],
 })
-export class GroupLogViewComponent implements OnChanges, OnDestroy {
+export class GroupLogViewComponent implements OnChanges, OnInit, OnDestroy {
 
   @Input() groupId?: string;
   @Input() showUserColumn = true;
 
+  @ViewChild('op') op?: OverlayPanel;
+  @ViewChildren('contentRef') contentRef?: QueryList<ElementRef<HTMLElement>>;
+
+  showOverlaySubscription?: Subscription;
+
   private readonly groupId$ = new ReplaySubject<string>(1);
   private readonly refresh$ = new Subject<void>();
   readonly state$ = this.groupId$.pipe(
     switchMap((groupId: string) => this.getData$(groupId)),
     mapToFetchState({ resetter: this.refresh$ }),
   );
+  private readonly showOverlaySubject$ = new BehaviorSubject<{ event: Event, itemId: string, target: HTMLElement }|undefined>(undefined);
+  showOverlay$ = this.showOverlaySubject$.asObservable().pipe(debounceTime(750), shareReplay(1));
 
   constructor(
     private activityLogService: ActivityLogService,
   ) {}
 
+  ngOnInit(): void {
+    this.showOverlaySubscription = this.showOverlay$.pipe(filter(isNotUndefined)).subscribe(data =>
+      this.op?.toggle(data.event, data.target)
```

Comment:
If you want your `mouseLeave` doing `this.showOverlaySubject$.next(undefined);` to have an effect, you do not have to filter out the undefined but just to `this.op?.show(...)` when not-undefined and `this.op?.hide()` when undefined.

Reply:
The idea is to consider also undefined values (thus removing `.pipe(filter(isNotUndefined))`) and that an undefined value _means_ "close overlay" while a defined value _means_ "open overlay", in code terms:
- if data is defined, call `this.op.show(data.event, data.target)` (does this method exist?)
- if data is undefined, call `this.op.hide()`

And since `undefined` values are handled here, to close the overlay we can do `this.showOverlaySubject$.next(undefined)` instead of `this.op.hide()` (in the `closeOverlaly` method)

Reply:
- On hover of items listed in "log view", display an overlay after a small delay.

`filter(isNotUndefined)` needs for that delay - `debounceTime(750)`. The improvement offered you means delay for close as well, which is not fit to the task

Reply:
See #1030 proposal